src/main/java/seedu/address/model/VersionedBudgetBook.java

```diff
@@ -0,0 +1,110 @@
+package seedu.address.model;
```

Comment:
Is there a need to keep a versioned budget book? I think it will only makes sense if can use "undo" and "redo" on it but I don't think "undo" and "redo" will work on within the budget book?

Reply:
not sure but i was thinking undo and redo is possible on the budget book

Reply:
is it possible now?